config/clusters/catalystproject-latam/common.values.yaml

```
@@ -12,134 +12,80 @@ jupyterhub:
-        slug: medium
-        profile_options:
-          requests:
-            # NOTE: Node share choices are in active development, see comment
-            #       next to profileList: above.
-            display_name: Node share
+        profile_options: &profile_options
+          resource_allocation: &resource_allocation
+            display_name: Resource Allocation
             choices:
-              mem_1:
-                display_name: ~1 GB, ~0.125 CPU
-                kubespawner_override:
-                  mem_guarantee: 0.903G
-                  cpu_guarantee: 0.117
-              mem_2:
-                display_name: ~2 GB, ~0.25 CPU
-                kubespawner_override:
-                  mem_guarantee: 1.805G
-                  cpu_guarantee: 0.234
-              mem_4:
+              mem_0_5:
                 default: true
-                display_name: ~4 GB, ~0.5 CPU
+                display_name: ~0.5 GB RAM, ~1/16 CPU
                 kubespawner_override:
-                  mem_guarantee: 3.611G
-                  cpu_guarantee: 0.469
-              mem_8:
-                display_name: ~8 GB, ~1.0 CPU
+                  mem_guarantee: 0.5G
+                  mem_limit: 1G
+                  cpu_guarantee: 0.05625
+                  cpu_limit: 1
+              mem_1_0:
+                display_name: ~1 GB RAM, ~1/8 CPU
                 kubespawner_override:
-                  mem_guarantee: 7.222G
-                  cpu_guarantee: 0.938
-              mem_16:
-                display_name: ~16 GB, ~2.0 CPU
+                  mem_guarantee: 1G
+                  mem_limit: 2G
+                  cpu_guarantee: 0.1125
+                  cpu_limit: 1
+              mem_2_0:
+                display_name: ~2 GB RAM, ~1/4 CPU
                 kubespawner_override:
-                  mem_guarantee: 14.444G
-                  cpu_guarantee: 1.875
-              mem_32:
-                display_name: ~32 GB, ~4.0 CPU
+                  mem_guarantee: 2G
+                  mem_limit: 4G
+                  cpu_guarantee: 0.225
+                  cpu_limit: 2
+              mem_4_0:
+                display_name: ~4 GB RAM, ~1/2 CPU
                 kubespawner_override:
-                  mem_guarantee: 28.887G
-                  cpu_guarantee: 3.75
-              mem_64:
-                display_name: ~64 GB, ~8.0 CPU
+                  mem_guarantee: 4G
+                  mem_limit: 8G
+                  cpu_guarantee: 0.45
+                  cpu_limit: 2
+              mem_8_0:
+                display_name: ~8 GB RAM, ~1.0 CPU
                 kubespawner_override:
-                  mem_guarantee: 57.775G
-                  cpu_guarantee: 7.5
-              mem_128:
-                display_name: ~128 GB, ~16.0 CPU
-                kubespawner_override:
-                  mem_guarantee: 115.549G
-                  cpu_guarantee: 15.0
+                  mem_guarantee: 8G
+                  mem_limit: 16GB
```

Checking the logs on the hub pod after the spawn failure, I could see the following error.

```
    traitlets.traitlets.TraitError: 16GB is not a valid memory specification. Must be an int or a string with suffix K, M, G, T
```

```suggestion
                  mem_limit: 16G
```